pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -127,12 +127,18 @@ sct_qc -i ${file_t2}.nii.gz -s ${file_t2}_label-SC_mask.nii.gz -d ${file_t2}_lab
 # Run sct_label_vertebrae for vertebral levels estimation
 label_if_does_not_exist ${file_t2}.nii.gz
 
+# Get centerline of the spinal cord
+sct_get_centerline -i ${file_t2}.nii.gz -c t2
+
+# Project the intervertebral disc labels to the spinal cord centerline
+sct_label_utils -i ${FILELABEL}.nii.gz -o ${FILELABEL}_proj.nii.gz -project-centerline ${file_t2}_centerline.nii.gz -qc ${PATH_QC} -qc-subject ${SUBJECT}
```

Comment:
I would use `_centerline.nii.gz`

```suggestion
sct_label_utils -i ${FILELABEL}.nii.gz -o ${FILELABEL}_centerline.nii.gz -project-centerline ${file_t2}_centerline.nii.gz -qc ${PATH_QC} -qc-subject ${SUBJECT}
```